.github/workflows/build_test.yml

```
@@ -73,6 +75,21 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - uses: ./.github/actions
 
+  without_opt:
+    needs: [smoke_test]
+    runs-on: ubuntu-latest
+    env:
+      WITHOUT_OPT: 1
```

Bikeshedding the name a bit, since OPT could be "options" or "optimizations"
```suggestion
      WITHOUT_OPTIMIZATIONS: 1
```

---

totally agree